src/index.js

```diff
@@ -34,11 +33,13 @@ module.exports = function(config) {
     LOGS_TOKEN: process.env.LOGENTRIES_TOKEN
   }, config);
 
-  //stores the cached required modules for the next requires on @dialonce/boot
+  // stores the cached required modules for the next requires on @dialonce/boot
+  /* eslint-disable global-require */
   initModule = {
     notifier: require('./bugsnag')(config.BUGS_TOKEN),
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this trailing comma. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.dialonce.io/coding_rules#rule_key=javascript%3ATrailingComma)